signup.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="utf-8">
+    <title>회원가입</title>
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css">
+    <link rel="stylesheet" href="styleLogin.css">
+  </head>
+  <body>
+    <div class="content">
+      <div class="head-img">
+        <a class="login-link" href="/">
+          <img class="pandaMarketImg" src="img/logo.png">
+        </a>
+      </div>
+      <div>
+        <p>이메일</p> 
+        <input type="email" name="userEmail" id="userName">
+        <p>닉네임</p> 
+        <input type="text" name="userName" id="nicname">
+        <p>비밀번호</p>
+        <input type="password" name="userPassword" id="userPassword">
+        <p>비밀번호 확인</p>
+        <input type="password" name="userPassword" id="userPassword">
```

Comment:
❗️ 수정요청
input의 name 속성은 해당 input값을 구분하는데 사용되는 속성으로 input의 경우 중복되면 안됩니다~
id 속성도 페이지당 유일해야하는 값이므로 두 input의 name, id값을 다르게 해주세요!

[input name attr](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name)

```suggestion
        <p>비밀번호</p>
        <input type="password" name="userPassword" id="userPassword">
        <p>비밀번호 확인</p>
        <input type="password" name="userPasswordCheck" id="userPasswordCheck">
```